### PR TITLE
Log response payload and headers in debug mode

### PR DESCRIPTION
**Pull request by erlendvollset**
## Description
Please describe the change you have made.

## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change. 
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring. 
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by simenl — APPROVED** [review]
LGTM - but someone with more python/sdk knowledge should review too

File: cognite/client/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-__version__ = "2.28.0"
+__version__ = "2.28.1"
```
Comment (me-ydv-5) [inline cognite/client/_version.py:1]:
```suggestion
__version__ = "2.27.1"
```

**Review by me-ydv-5 — APPROVED** [review]
lgtm

File: cognite/client/_version.py (hunk 1 above)
Comment (me-ydv-5) [reply cognite/client/_version.py:1]: ah no, 2.28 is already there

Comment (erlendvollset) [reply cognite/client/_version.py:1]: 2.28 already published, just changelog wasn't updated.

**Pull request merged**